i3pystatus/pulseaudio/__init__.py

```diff
@@ -197,22 +197,23 @@ def sink_info_cb(self, context, sink_info_p, eol, _):
             else:
                 raise Exception("bar_type must be 'vertical' or 'horizontal'")
 
-            selected = ""
-            dump = subprocess.check_output("pacmd dump".split(), universal_newlines=True)
-            for line in dump.split("\n"):
-                if line.startswith("set-default-sink"):
-                    default_sink = line.split()[1]
-                    if default_sink == self.current_sink:
-                        selected = self.format_selected
+            # TODO: Looks like we can't access daemon config using pactl.
+            # Remove this part if that's true or find an appropriate pactl alternative.
+            # selected = ""
+            # dump = subprocess.check_output("pacmd dump".split(), universal_newlines=True)
+            # for line in dump.split("\n"):
+            #     if line.startswith("set-default-sink"):
+            #         default_sink = line.split()[1]
+            #         if default_sink == self.current_sink:
+            #             selected = self.format_selected
```

Comment:
I'm pretty sure that this issue can be easily be resolved with the following:
```
            selected = ""
            default_sink = subprocess.check_output("pactl get-default-sink".split(), universal_newlines=True).strip()
            if default_sink == self.current_sink:
                selected = self.format_selected
```

Reply:
I have tested with a full pulseaudio and a full pipewire setup, and this patch works.

`pactl get-default-sink` and `pacmd dump | grep set-default-sink | awk '{print $2}'` (the equivalent bash command of the current code) gives the same outputs on a full pulseaudio setup.